### PR TITLE
Implemented shoreline forecasts line and polygons

### PR DESCRIPTION
**Pull request by zxue-usgs**
Added forecast line and polygon sld generation using categorical_line and catagorical_polygon jsp

fixed a few documentations on authors
updated attribute test to include new attributes

### CODE REVIEW

File: coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/sld/ForecastUncy.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+//
+//Polygon outline thickness:  1.0
+//Polygon outline color:  R G B all 110
+//Polygon transparency:  50%  (in Arc, rt click on shp layer, Properties - Display)
+//
+//10-yr line:  R 1, G 133, B 113 #018571
+//10-yr polygon:  R 128, G 205, B 193 #80CDC1
+//
+//20-yr line:  R 166, G 97, B 26 #A6601A
+//20-yr polygon:  R 223, G 194, B 125 #DFC27D
```
Comment (zmoore-usgs) [inline coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/sld/ForecastUncy.java:25]: ^ Feel free to keep the above comments if you want for future reference, though I would recommend changing them to a block comment with a bit of context about what each thing refers to (I.E: The line referring to the other Forescast SLD)

**Review by zmoore-usgs — COMMENTED** [review]
Overall these changes look good! One thing I'd take a look at though is whether or not the click-to-identify is working as you would expect it to. For my categorical stuff to show up properly I had to add the new attribute I added to the list here: 

https://github.com/USGS-CIDA/coastal-hazards/blob/master/coastal-hazards-portal/src/main/webapp/js/cch/objects/LayerIdentifyControl.js#L520

File: coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/sld/ForecastUncy.java (hunk 1 above)
Comment (zxue-usgs) [reply coastal-hazards-portal/src/main/java/gov/usgs/cida/coastalhazards/sld/ForecastUncy.java:25]: Thanks for catching that, I deleted these from the other file but forgot about this one.

**Comment by zxue-usgs** [comment]
Codacy is not happy about the "Forecast" and "ForecastUncy" classes

**Comment by zmoore-usgs** [comment]
> Codacy is not happy about the "Forecast" and "ForecastUncy" classes

Yeah, I wouldn't worry about that too much. You could change the `Attributes.*` import to only import the attribute the SLD is using like `Attribute.FORECAST`.

As far as the class name, it wants the word "Utility" (or one of the other it listed) in the class name since it's under a path called "util" but that's not a naming convention this project uses so I wouldn't worry about that one.

**Comment by zxue-usgs** [comment]
> Overall these changes look good! One thing I'd take a look at though is whether or not the click-to-identify is working as you would expect it to. For my categorical stuff to show up properly I had to add the new attribute I added to the list here:
> 
> https://github.com/USGS-CIDA/coastal-hazards/blob/master/coastal-hazards-portal/src/main/webapp/js/cch/objects/LayerIdentifyControl.js#L520

Thanks for the heads up, click-to-identify does not work. I will look into how we want it to be displayed. Holding off on this PR for now.

**Comment by zxue-usgs** [comment]
I don't understand why travis failed.

**Comment by zmoore-usgs** [comment]
Huh, yeah that's a weird one.. I'm thinking Travis might've gotten an error page trying to pull some dependencies (hence the `<html>` bit it mentioned not recognizing). Going to try to manually kick it off again

**Pull request merged**